packages/react-ai/src/index.ts

```diff
@@ -0,0 +1,2 @@
+import { createAIConversation } from './components/AIConversation';
```

Comment:
Shouldn't there be a package.json file that sets this package as private?

Reply:
added private flag